docs/dev/engineering/ci.md

```diff
@@ -1,15 +1,3 @@
-<!-- START doctoc generated TOC please keep comment here to allow auto update -->
-<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-
-**Contents**
-
-- [Continuous Integration](#continuous-integration)
-  - [CircleCI](#circleci)
-    - [Run Circle CI locally (requires Docker)](#run-circle-ci-locally-requires-docker)
-    - [Installing artifacts created by CircleCI](#installing-artifacts-created-by-circleci)
-
-<!-- END doctoc generated TOC please keep comment here to allow auto update -->
-
 # Continuous Integration
 
 [Circle CI](https://circleci.com/) is used for continuous integration.
```

Comment:
Found `Configured via ./.circleci/config.yml` in https://github.com/motin/firefox-translations/blob/improve-docs-and-clean-up/docs/dev/engineering/ci.md#circleci. Is it intended?

Reply:
Yes, that line explains by what file CircleCi is configured. The header above it can be removed though (https://github.com/mozilla-extensions/firefox-translations/pull/250/files/8b8df5ecee548d0b1f777c19cc2ff7fab64eca3d#diff-e135dc30b14b335235406f940b33c2da8b3c0252ccd31d67a91196edf5db86d5R5)

Reply:
I meant the path of `config.yml` relative to `ci.md` is wrong. It should be `../../../.circleci/config.yml`.  Is it by mistake or am I missing something here?

Reply:
I'll make it clearer that the path (which is not a link) is relative to the root of the repository.